master/master.cfg

```diff
@@ -249,6 +251,15 @@ class PGOUnixBuild(NonDebugUnixBuild):
     factory_tags = ['pgo']
 
 
+class ClangUBSanLinuxBuild(UnixBuild):
+    configureEnv = UnixBuild.configureEnv.copy()
```

Comment:
You might use dict(**UnixBuild.configureEnv, CC='clang', ...) but I'm not sure it's more readable.

Reply:
Alternately, these could be passed in `configureFlags` (`'CC=clang', 'LD=clang', ...`), just to not add `configureEnv`.  I'm fine with giving this a shot as is, though.

Reply:
I was just replicating what I did on the command line that had worked.  if they work as parameters, I'll do that, I'll double check here and simplify to just use the params if so...